aten/src/THC/CMakeLists.txt

```diff
@@ -51,7 +51,6 @@ set(ATen_CUDA_SRCS ${ATen_CUDA_SRCS}
   ${CMAKE_CURRENT_SOURCE_DIR}/THCTensorMathPairwise.cu
   ${CMAKE_CURRENT_SOURCE_DIR}/THCTensorMathReduce.cu
   ${CMAKE_CURRENT_SOURCE_DIR}/THCTensorScatterGather.cu
-  ${CMAKE_CURRENT_SOURCE_DIR}/THCTensorSort.cu
   ${CMAKE_CURRENT_SOURCE_DIR}/THCSortUtils.cu
```

Comment:
Is `THCSortUtils.cu` still used?

Reply:
I think it is still used for a few things like LTComp/GTComp...
Thanks for pointing this out because I think we may be able to delete the rest of the functions here.